lib/riax.ex

```diff
@@ -17,7 +18,21 @@ defmodule Riax do
   Retrieve keys
   """
   def keys() do
-    sync_command(:key, :keys)
```

Comment:
you could do without these one-liners

Reply:
How so? 🤔 
Do you mean to use only coverage_command? I thought It would be clearer to do it this way

Reply:
these are all pass-through method, so they add little value, and they don't improve readability, imo they actually worsen it since you are losing information. when the someone reads `coverage_command(:keys)` at the calling place, they get a much rich idea of what's going on than if they read `keys()`

(I'm ousterhouting here)

Reply:
I think you're right about this, `coverage_command` alone is the right approach here.
More so, these comments make me think about what the Riax, and the library as a whole, should be doing. 🤔 
I'm starting to think that the Riax module, when offered as a dependency, should be a way to only give commands to a particular VNode implementation.

Reply:
yes I think that makes more sense. That being said, you don't need to figure out in advance, you can proceed with solving your concrete problem and postpone the API decision, then flesh out the API when you have the concrete use case to use as a reference
 (this is the way I tend to design APIs, it doesn't mean that you need to do the same; some people prefer to think about the abstract first, but I think by doing that in this case you'd risk moving in the wrong direction and waste your efforts)